sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -290,6 +290,27 @@ private[ui] class ExecutionPagedTable(
         }
       }
 
+    var headerNameWithTooltips: Map[String, String] = Map(
+      "ID" -> "Execution ID of the SQL query.",
+      "Description" -> "The description links to the query details page",
+      "Submitted" -> "Submitted",
```

Comment:
This doesn't add anything

Reply:
@srowen, any suggestion to show tooltips msg